plugins/org.eclipse.elk.core.debug/src/org/eclipse/elk/core/debug/views/graph/LayoutGraphViewDropAdapter.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.elk.core.debug.views.graph;
+
+import org.eclipse.jface.viewers.Viewer;
+import org.eclipse.jface.viewers.ViewerDropAdapter;
+import org.eclipse.swt.dnd.FileTransfer;
+import org.eclipse.swt.dnd.TransferData;
+
+/**
+ * This adapter provides a drop support for the tree viewer of the Layout Graph view.
+ * 
+ * @author <a href="mailto:laurent.redor@obeo.fr">Laurent Redor</a>
```

Comment:
Please remove the author tag. We're not using those since the Git history shows everything one would want to know. :)

Also, remove the empty lines please.

Reply:
done